example/example.dart

```diff
@@ -35,6 +40,9 @@ void main() {
   });
 
   test("Let's verify some behaviour!", () {
+    // Stub a method before interacting with it.
+    when(cat.sound()).thenReturn('Meow');
```

Comment:
Should I use the `MockCatRelaxed` overall to reduce changes?

Reply:
No I love it. Not-relaxed is the new way! 😁